Todo.py

```diff
@@ -0,0 +1,22 @@
+class Todo():
+    def __init__(self, name, owner, is_important=False, is_done=False):
+        self.name = name
+        self.owner = owner
+        self.is_important = is_important
+        self.is_done = is_done
+
+    def toggle_is_important(self):
+        self.is_important = not self.is_important
+
+    def toggle_is_done(self):
```

Comment:
Could you try rename according our conventions? e.g. `ToggleIsDone`

Reply:
done, thanks